ftdetect/wgsl.vim

```diff
@@ -0,0 +1 @@
+au BufRead,BufNewFile *.wgsl set filetype=wgsl
```

Comment:
Please remove that; nvim-treesitter no longer acts as a filetype plugin. You should make a PR adding this filetype to Vim, which will then be ported to Neovim. This way, other people will profit from this, too.